### ISSUE TITLE
Add Search API Spellcheck ("Did you mean ...?") module to LGD core

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov#603]
On the Essex Send project, we added the [Search API Spellcheck](https://www.drupal.org/project/search_api_spellcheck) module, so if someone enters "autsim" the search result will suggest "Did you mean 'autism'?". 

I think this is a great accessibility improvement for people whose reading age may be low, or may be don't have English as a first language, or may have dyslexia, etc. But it is something that I think all of our websites could benefit from, not just SEND websites.

**Comment by willguv** [comment]
This would be great to add @markconroy 

If you have time, could you do a PR? Many thanks!

Copying @stephen-cox

**Comment by finnlewis** [comment]
Plus one from me! Should we put this in localgov_search, localgov_directories or both ? 

Copying @ekes

**Comment by ekes** [comment]
Sure we've discussed this before, but maybe it was just in slack. It's a solr thing. So it would be config and dependency on the appropriate solr modules.

**Comment by willguv** [comment]
Oh, does this mean we can't easily progress it?

**Comment by ekes** [comment]
We now have solr backend modules as well as db backend modules. So the point was it should be added as config and dependency for the solr backend module (when Finn was asking where to put it).
